e2e/rbd_helper.go

```diff
@@ -1063,35 +1063,59 @@ func waitToRemoveImagesFromTrash(f *framework.Framework, poolName string, t int)
 
 // imageInfo strongly typed JSON spec for image info.
 type imageInfo struct {
-	Name        string `json:"name"`
-	StripeUnit  int    `json:"stripe_unit"`
-	StripeCount int    `json:"stripe_count"`
-	ObjectSize  int    `json:"object_size"`
+	Name            string   `json:"name"`
+	ID              string   `json:"id"`
+	Size            int64    `json:"size"`
+	Objects         int      `json:"objects"`
+	Order           int      `json:"order"`
+	ObjectSize      int      `json:"object_size"`
+	SnapshotCount   int      `json:"snapshot_count"`
+	BlockNamePrefix string   `json:"block_name_prefix"`
+	Format          int      `json:"format"`
+	Features        []string `json:"features"`
+	OpFeatures      []string `json:"op_features"`
+	Flags           []string `json:"flags"`
+	CreateTimestamp string   `json:"create_timestamp"`
+	AccessTimestamp string   `json:"access_timestamp"`
+	ModifyTimestamp string   `json:"modify_timestamp"`
+	StripeUnit      int      `json:"stripe_unit"`
+	StripeCount     int      `json:"stripe_count"`
```

Comment:
@OdedViner  i dont think we need all the fields in this struct unless we are using it, as we are returning the string as a output we are good for logging. please revert this back

Reply:
done